artichoke-backend/src/extn/core/kernel/require.rs

```diff
@@ -162,4 +166,65 @@ pub mod method {
         }
         Err(Error::CannotLoad(args.file.to_owned()))
     }
+
+    fn load_impl(interp: &Artichoke, args: Args, base: &str) -> Result<Require, Error> {
+        let interp = interp.clone();
```

Comment:
I think this clone is not needed. (I know it is there in `require`. I think that is also a bug.)

Reply:
Removed.